app/src/main/res/layout/news_feed_fragment.xml

```diff
@@ -7,6 +7,17 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <ProgressBar
```

Comment:
Is it shown with all the other views? By the idea, it should replace all the other views and appear at the center.